src/Egret.Cli/Serialization/Audacity/AudacityImporter.cs

```diff
@@ -0,0 +1,118 @@
+
+                await using Stream stream = fileSystem.File.OpenRead(path);
+                var dataFile = serializer.Deserialize(stream, path);
+
+                var expectations = dataFile?.Tracks?.SelectMany((track, trackIndex) =>
+                        track.Labels.Select((label, labelIndex) =>
+                        {
+                            var temporalTolerance = context.Include.TemporalTolerance ?? defaultTolerance;
+                            var spectralTolerance = context.Include.SpectralTolerance ?? defaultTolerance;
+                            var overrideBounds = context.Include.Override;
+
+                            var name =
+                                $"Audacity Label {label.Title} in track {track.Name} ({trackIndex}:{labelIndex})";
+
+                            return label.IsSelPoint
+                                ? (IExpectation)new TemporalExpectation(label)
+                                {
+                                    Time = new TimeRange(
+                                        label.TimeStart.WithTolerance(temporalTolerance),
+                                        label.TimeEnd.WithTolerance(temporalTolerance)),
+                                    AnyLabel = new[] {label.Title},
+                                    Name = name,
+                                }
+                                : (IExpectation)new BoundedExpectation(label)
+                                {
+                                    Bounds = new Bounds(
+                                        overrideBounds?.Start ?? label.TimeStart.WithTolerance(temporalTolerance),
+                                        overrideBounds?.End ?? label.TimeEnd.WithTolerance(temporalTolerance),
+                                        overrideBounds?.Low ?? label.SelLow.WithTolerance(spectralTolerance),
+                                        overrideBounds?.High ?? label.SelHigh.WithTolerance(spectralTolerance)),
+                                    AnyLabel = new[] {label.Title},
+                                    Name = name,
+                                };
+                        }))
+                    .ToList() ?? new List<IExpectation> {new NoEvents()};
+
+                if (expectations.Count == 1 && expectations[0] is NoEvents)
+                {
+                    logger.LogWarning("No annotations found in {path}, producing a no events expectation", path);
+                }
+
+                if (context.Include.Exhaustive.HasValue && context.Include.Exhaustive.Value)
+                {
+                    expectations.Add(new NoExtraResultsExpectation() {Match = context.Include.Exhaustive.Value});
+                }
+
+                logger.LogTrace("Data file converted to expectations: {@expectations}", expectations);
+
+                yield return new TestCase {SourceInfo = dataFile?.SourceInfo, Expect = expectations.ToArray(),};
```

Comment:
dataFile should never be null right? If it is, then we should fall back to providing some source info (like the `path`) so in error cases at least we can print a message that point to the right source  file.